src/main/java/frc/robot/Constants.java

```diff
@@ -15,7 +18,20 @@ public static class OI {
     public static final int JOYSTICK_PORT = 0;
   }
 
+  public static class Landmarks {
+    // Landmarks on the Blue side can be reflected to show the respective locations on the Blue side
+    public static final Pose2d RIGHT_NOTE_LOCATION = new Pose2d(2.5, 4.1, new Rotation2d());
```

Comment:
Should these be renamed to AMP_SIDE_NOTE / STAGE_SIDE_NOTE to reflect auton chooser changes?